src/main/java/io/github/thebusybiscuit/slimefun4/implementation/listeners/MobDropListener.java

```diff
@@ -34,7 +36,17 @@ public void onEntityKill(EntityDeathEvent e) {
 
             Set<ItemStack> customDrops = SlimefunPlugin.getRegistry().getMobDrops(e.getEntityType());
             if (customDrops != null && !customDrops.isEmpty()) {
-                addDrops(p, customDrops, e.getDrops());
+            	for(ItemStack is:customDrops)
+            	{
+            		SlimefunItem sfi = SlimefunItem.getByItem(is);
+            		if(sfi != null && sfi instanceof ChanceDrop) {
+            			if(((ChanceDrop)sfi).getChance() <= ThreadLocalRandom.current().nextInt(0,100)) 
```

Comment:
Should make a variable for the random since it could be called multiple times and this isn't exactly the least expensive thing in the world

Reply:
No need to specify the 0 as origin